spec/spec_helper.rb

```diff
@@ -17,7 +17,7 @@
 require 'pry'
 
 # Requires supporting ruby files with custom matchers and macros, etc, in spec/support/ and its subdirectories.
-Dir[File.dirname(__FILE__) + '/support/**/*.rb'].each { |file| require file }
+Dir[File.dirname(__FILE__) + '/support/**/*.rb'].sort.each { |file| require file }
```

Comment:
Why is this necessary?

Reply:
If you refer to the commit log, this was changed/fixed as Rubocop flagged it during the TravisCI build (which weirdly should already be failing in master, as my changes didn't affect any of the Rubocop flagged offenses 🤷‍♂ ). 

```ruby
spec/spec_helper.rb:20:1: W: Lint/NonDeterministicRequireOrder: Sort files before requiring them.
Dir[File.dirname(__FILE__) + '/support/**/*.rb'].each { |file| require file }
```
More info on this [here](https://rubocop.readthedocs.io/en/latest/cops_lint/#lintnondeterministicrequireorder).